src/coreclr/pal/src/misc/sysinfo.cpp

```diff
@@ -613,9 +613,17 @@ PAL_GetLogicalProcessorCacheSizeFromOS()
     {
         int64_t cacheSizeFromSysctl = 0;
         size_t sz = sizeof(cacheSizeFromSysctl);
-        const bool success = sysctlbyname("hw.l3cachesize", &cacheSizeFromSysctl, &sz, nullptr, 0) == 0
+        const bool success = false
```

Comment:
We should also make the same fix here: https://github.com/dotnet/runtime/blob/856e8c3510cb0aaaa3b1a1679552c03ae5308b9e/src/coreclr/gc/unix/gcenv.unix.cpp#L951

Reply:
done!